cs/src/comm/epoxy-transport/EpoxyTlsConfig.cs

```diff
@@ -0,0 +1,187 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Bond.Comm.Epoxy
+{
+    using System;
+    using System.Net.Security;
+    using System.Security.Authentication;
+    using System.Security.Cryptography.X509Certificates;
+
+    /// <summary>
+    /// Represents join client- and server-side TLS options.
```

Comment:
joint? Either way, how about "represents TLS options common to client and server configurations"?